.travis.yml

```diff
@@ -46,6 +46,7 @@ install:
 before_script:
     - if [ "$CHECK_DOC" = 1 -a "$TRAVIS_EVENT_TYPE" = "pull_request" ]; then contrib/devtools/commit-script-check.sh $TRAVIS_COMMIT_RANGE; fi
     - if [ "$CHECK_DOC" = 1 ]; then contrib/devtools/check-doc.py; fi
+    - if [ "$CHECK_DOC" = 1 ]; then contrib/devtools/lint-all.sh; fi
```

Comment:
This needs to be guarded by `TRAVIS_COMMIT_RANGE` if you fail on `-z $TRAVIS_COMMIT_RANGE`, no?

Reply:
I'm not even sure if I should have changed it to fail, should it just exit quietly if there's no commit range?

Reply:
Actually this should be fine, as `TRAVIS_COMMIT_RANGE` should never be empty according to the travis doc. (It is only empty for branches with a single commit, which we don't have)